frontend/apps/crates/components/src/module/_common/play/entry/ending/dom.rs

```diff
@@ -18,7 +18,7 @@ impl Ending {
 
         if let Some(kind) = state.kind {
             match kind {
-                ModuleEnding::Next => {
```

Comment:
Resolves an issue when the ending is `Positive`, the activity does not move on to the next activity.